packages/cli/src/workflow-runner.ts

```diff
@@ -66,10 +66,15 @@ export class WorkflowRunner {
 		//
 		// FIXME: This is a quick fix. The proper fix would be to not remove
 		// the execution from the active executions while it's still running.
-		if (error instanceof ExecutionNotFoundError || error instanceof ExecutionCancelledError) {
+		if (
+			error instanceof ExecutionNotFoundError ||
+			error instanceof ExecutionCancelledError ||
+			error.message.includes('cancelled')
```

Comment:
Not ideal, but see: https://linear.app/n8n/issue/PAY-2414/cancellation-in-scaling-mode-leads-to-job-being-re-enqueued#comment-f257b343